webrtc.html

```diff
@@ -1301,6 +1301,14 @@
                     <code>InvalidModificationError</code> and abort these steps.
                   </li>
                   <li>
+                    <p>If <var>description</var> is set as a local description,
+                    if <code><var>description</var>.type</code> is <code>"rollback"</code>
+                    and <a>signaling state</a> is not <code>"have-local-offer"</code>
```

Comment:
Maybe *"is neither "have-local-offer" nor "have-local-pranswer""* ?

I'm not a JSEP expert, but [section 4.1.8.2.](https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-20#section-4.1.8.2) seems to imply rollback is allowed from any state but stable.